FirebaseRemoteConfig/Sources/RCNPersonalization.m

```diff
@@ -37,16 +38,31 @@ - (void)logArmActive:(NSString *)key config:(NSDictionary *)config {
   }
 
   NSDictionary *metadata = ids[key];
-  if (!metadata || metadata[kPersonalizationId] == nil) {
+  NSString *personalizationId = metadata[kPersonalizationId];
```

Comment:
move this to after the `!metadata` check ?

Reply:
Done.